build/config/SignToolData.json

```diff
@@ -5,50 +5,545 @@
       "strongName": "MsSharedLib72",
       "values": [
         "Dlls\\BasicCodeAnalysis\\Microsoft.CodeAnalysis.VisualBasic.dll",
+        "Dlls\\BasicCodeAnalysis\\cs\\Microsoft.CodeAnalysis.VisualBasic.resources.dll",
+        "Dlls\\BasicCodeAnalysis\\de\\Microsoft.CodeAnalysis.VisualBasic.resources.dll",
+        "Dlls\\BasicCodeAnalysis\\es\\Microsoft.CodeAnalysis.VisualBasic.resources.dll",
```

Comment:
Can we use wildcards here as in other repos? Keeps the file at a managable size. Wild cards were actually specifically added for this case.

Reply:
Ah, I didn't know wildcards were an option. Will do.

Reply:
@jaredpar Will a `build.cmd -sign` exercise the data in this file? So I can tell if I've screwed something up or missed a file before we try to run it through a real signed build?

Reply:
Yes. It's in fact designed to do exactly that. Run the following

> build.cmd -buildall -sign